src/framework/core/js/Fluid.js

```diff
@@ -32,7 +32,7 @@ var fluid = fluid || fluid_2_0;
 (function ($, fluid) {
     "use strict";
 
-    fluid.version = "Infusion 2.0-SNAPSHOT";
+    fluid.version = "Infusion 2.0-dev";
```

Comment:
I think it probably should, yes. Is there any reason not to? If I remember, this is just a helpful human-readable string, right? If, so, we might as well make it consistent.